organize source files in GPSBabel.pro, CMakeLists.text`

tsteven4 opened the pull request:
The files are listed one per line in alphabetical order.  This makes it much easier to compare the two.  It also makes it easier to see what happened should the reaper return.

tsteven4 committed:
organize files in GPSBabel.pro and CMakeLists.tst

This makes it easier to compare the two, and easier to see what
changed when nuking things.

tsteven4 committed:
fix for lack of continuation line

robertlipe reviewed (approved):
Yes!, Please!
I've needed to do that for years.
If you have time, please change the remove_word to remove_line calla in tools/nuke_format
If not, that's still fine.
Thanks!

tsteven4 committed:
update nuke_format for sources on their own lines.

delete MSVC project and solution files.  The project files should
be generated with "qmake -tp vc GPSBabel.pro".  MSVC will create
the solution file if needed.
